### PR TITLE
[1LP][WIPTEST] Widgetastic Slider widget

### PR DESCRIPTION
**Pull request by izapolsk**
Purpose or Intent
=================
one small Slider widget 

I've checked all the methods manually.

### CODE REVIEW

File: widgetastic_manageiq.py [hunk 1]
```diff
@@ -1309,3 +1309,46 @@ def all_emails(self):
...
+    def min(self):
+        return float(self.browser.get_attribute('min', self))
+
+    @property
+    def step(self):
+        return float(self.browser.get_attribute('step', self))
+
+    @value.setter
+    def value(self, val):
+        self.browser.set_attribute('value', val, self)
```
Comment (otsuman) [inline widgetastic_manageiq.py]: **REQUIRED**: This setter doesn't work. It will not change "value" field in GUI. It must be changed

Comment (izapolsk) [inline widgetastic_manageiq.py]: it changes value in gui but javascript doesn't pick up changes for some reason.

Comment (otsuman) [inline widgetastic_manageiq.py]: I checked same control in Bottlenecks page. It works exactly like this one.

File: widgetastic_manageiq.py [hunk 2]
```diff
@@ -1309,3 +1309,46 @@ def all_emails(self):
...
+        self.value += self.step
+
+    def decrease(self):
+        self.value -= self.step
+
+    def read(self):
+        return self.value
+
+    def fill(self, val):
+        self.value = val
```
Comment (mfalesni) [inline widgetastic_manageiq.py]: **REQUIRED**: fill must return true or false.

**Comment by izapolsk** [comment]
It seems this control appears only along with timelines chart. Additionally, it is implemented in strange way.
So, I'm closing this PR and it will come along with timelines view.

**Pull request closed without merging**